assets/cluster-monitoring-operator/cluster-role.yaml

```diff
@@ -9,3 +9,9 @@ rules:
   - namespaces
   verbs:
   - get
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - alertmanagers
+  verbs:
+  - get
```

Comment:
The SubjectAccessReview is only against the `openshift-monitoring` namespace: https://github.com/openshift/cluster-monitoring-operator/pull/502/files#diff-c4e970fc1be688ce39664190175bfd86

Reply:
I think we cannot express that in a cluster role. That cluster role can be bound to a (namespaced) role limiting it, but afaik we cannot express such behavior here.

Alternative: Introduce a dedicated `alertmanager-view` Role (not ClusterRole) which has to be explicitly bound to the grantee.

/cc @cblecker: can you give more insight/direction on what path you wanted to go with this?

Reply:
I think `alertmanager-access` was previously suggested. I think that's the right way to go. It might be slightly breaking, but I think it's for the better.

Reply:
Yes, I had created a `alertmanager-access` role for this purpose in my last PR: https://github.com/openshift/cluster-monitoring-operator/pull/502/files#diff-8a39df569f081fc563dde843fae1075e

The thing is, because this is looking at a CR, I don't think we expect or will act on instances of the CR that are created outside of the openshift-monitoring namespace, will we? I think in effect, it won't make a difference. I'm also okay changing the SAR to be a cluster-wide one (removing the namespace specifically) and make it more clear.. again, because we're looking at a custom resource (and not `secret get`) I think there is flexibility in how we express this.